src/lex.js

```diff
@@ -435,6 +435,18 @@ Lexer.prototype = {
           body = body.substr(str.length + 1);
         }
 
+        // To handle rarer case when special word is separated from label by
+        // multiple spaces or tabs
+        var strIndex = body.indexOf(str);
+        if (!isSpecial && strIndex >= 0 && body.charAt(strIndex + str.length + 1) === " ") {
+          var specialSubstr = body.substr(0, strIndex + str.length);
+          var isAllWhitespace = (specialSubstr.length - specialSubstr.trim().length) === strIndex;
```

Comment:
I think this can be simplified by capturing just the preceding characters (`body.substr(0, strIndex)`) and making sure that string contains only whitespace (as in `.trim().length === 0`).